api_tests/tests/reasonable_adjustments/test_happy_cases.py

```diff
@@ -261,6 +261,23 @@ def test_remove_ra_record_post(self):
         # Then
         assert_that(expected_status_code).is_equal_to(response.status_code)
 
+    @pytest.mark.happy_path
+    @pytest.mark.integration
```

Comment:
I think we use `smoke` as integration test marker. This is the first time I'm seeing this marker :)

Reply:
the markers `smoke` and `integration` were added as part of AMB-359, they should be in `pytest.ini`. The pipeline runs all tests marked as integration.